STYLEGUIDE.md

```diff
@@ -1,40 +1,125 @@
-# Auth0 Style Guide #
+# Auth0 Style Guide
 
 This style guide will cover the terminology and content specific to Auth0, along with some comments on common writing issues.
+
 For general software-industry styles and terminology, see the [Microsoft Manual of Style](https://eucalyptus.atlassian.net/wiki/download/attachments/76611622/microsoft_manual_of_style_fourth_edition.pdf?version=2&modificationDate=1424379604164&api=v2).
 
 ## Voice
 
-It is important to speak to the reader directly, to engage in a conversation with the reader. This is the most effective way to communicate. In order to do this, address the reader as "you." Do not use such indirect language as "one" or other passive ways of addressing the user.
+It is important to speak to the reader directly, to engage in a conversation with the reader. This is the most effective way to communicate. In order to do this, address the reader as "you." Do not use such indirect language as "one" or other passive ways of addressing the user. Avoid "we".
```

Comment:
"You should speak to the reader directly, so that your document reads as if you're engaged in a conversation with the reader..."